application/src/main/java/org/thingsboard/server/service/transport/DefaultTransportApiService.java

```diff
@@ -226,6 +242,52 @@ private ListenableFuture<TransportApiResponseMsg> validateCredentials(TransportP
         }
     }
 
+    protected ListenableFuture<TransportApiResponseMsg> validateOrCreateDeviceX509Certificate(String certChain, DeviceCredentialsType credentialsType) {
+        List<String> chain = convertX509CertificateChainToList(certChain);
+        String updateDeviceCertificateValue = chain.get(0);
+        String updateDeviceCertificateHash = EncryptionUtil.getSha3Hash(updateDeviceCertificateValue);
+        String deviceCommonName = "";
+        try {
+            deviceCommonName = SslUtil.parseCommonName(readCertFile(chain.get(0)));
+        } catch (Exception ignored) {
```

Comment:
Why do you prefer to ignore this exception? 
I believe in real case production that includes a lot of potential points of failure, it's important quickly understand where is the root cause of the problem and warn message in the core engine that certificate cannot be parsed or common name cannot be obtain could help on this.

Reply:
This error could occur if we cannot read and create an X509 object from String, so I am not sure it's the right warning message.